frontend/src/hooks/useExportCsvEffect.ts

```diff
@@ -2,26 +2,35 @@ import { useState } from 'react'
 import { CSVReportRequestDTO } from '@src/clients/report/dto/request'
 import { csvClient } from '@src/clients/report/CSVClient'
 import { ERROR_MESSAGE_TIME_DURATION } from '@src/constants'
+import { NotFoundException } from '@src/exceptions/NotFoundException'
 
 export interface useExportCsvEffectInterface {
   fetchExportData: (params: CSVReportRequestDTO) => void
   errorMessage: string
+  isExpired: boolean
 }
 
 export const useExportCsvEffect = (): useExportCsvEffectInterface => {
   const [errorMessage, setErrorMessage] = useState('')
+  const [isExpired, setIsExpired] = useState(false)
 
   const fetchExportData = async (params: CSVReportRequestDTO) => {
     try {
+      setIsExpired(false)
       return await csvClient.exportCSVData(params)
     } catch (e) {
       const err = e as Error
-      setErrorMessage(`failed to export csv: ${err.message}`)
-      setTimeout(() => {
-        setErrorMessage('')
-      }, ERROR_MESSAGE_TIME_DURATION)
+      if (err instanceof NotFoundException) {
+        setIsExpired(true)
+        setErrorMessage('csv file expired')
+      } else {
+        setErrorMessage(`failed to export csv: ${err.message}`)
```

Comment:
update to `Failed to export the CSV file, reason: `

Reply:
I delete setErrorMessage(`The CSV file has been expired : ${err.message}`) this line , 
because the expired dialog already hint the message . If ErrorMessage still exist ,it will be like this:
<img width="568" alt="Screenshot 2023-11-16 at 11 26 27" src="https://github.com/au-heartbeat/Heartbeat/assets/74648528/85b857e9-1df6-4872-8d53-f29245ecda34">